docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -70,6 +70,7 @@ Some features are enabled by default. You can disable these feature by setting t
 | `autoMigratePiechartPanel`             | Migrate old piechart panel to supported piechart panel - broken out from autoMigrateOldPanels to enable granular tracking                                                                    |
 | `autoMigrateWorldmapPanel`             | Migrate old worldmap panel to supported geomap panel - broken out from autoMigrateOldPanels to enable granular tracking                                                                      |
 | `autoMigrateStatPanel`                 | Migrate old stat panel to supported stat panel - broken out from autoMigrateOldPanels to enable granular tracking                                                                            |
+| `autoMigrateXYChartPanel`              | Migrate old XYChart panel to new XYChart2 model                                                                                                                                              |
```

Comment:
not currently hooked up. will work on this during bug bash

Reply:
I think i just fixed it with latest commit